user_tools/src/spark_rapids_pytools/resources/dataproc-configs.json

```diff
@@ -258,6 +258,46 @@
       ]
     }
   },
+  "clusterInference": {
+    "defaultClusterConfig": {
+      "clusterName": "default-cluster-prop",
+      "clusterUuid": "1234-5678-1234567",
+      "config": {
+        "gceClusterConfig": {
+          "zoneUri": "us-central1-a"
+        },
+        "masterConfig": {
+          "instanceNames": [
+            "test-node"
+          ],
+          "machineTypeUri": "n1-standard-16",
+          "numInstances": 1
+        },
+        "workerConfig": {
+          "instanceNames": [
+            "test-node"
+          ],
+          "machineTypeUri": "n1-standard-32",
+          "numInstances": 1
+        }
+      },
+      "status": {
+        "state": "STOPPED"
+      }
+    },
```

Comment:
Is this the purpose of this is to serve as a template? If so, then we can consider using moustache.
I added some mechanism to use templates before. you can find the template files in `resources/templates`

Reply:
Yes. We want to use these as templates. I will look into using `chevron` and `moustache`

Reply:
Moved to using mustache